packages/server/src/service/mg.ts

```diff
@@ -85,3 +86,35 @@ export const sendSignupEmail = async (
   const emailData = toSignupEmailData(letter, voterEmail, officialEmails, { pdfBuffer, force })
   return mg.messages().send(emailData)
 }
+
+const mailgunToGCPLogLevels = {
+  "info": "INFO",
+  "warn": "WARNING",
+  "temporary": "WARNING",
+  "error": "ERROR"
+}
+export const logMailgunLogToGCP = async (request: Object): Promise<void> => {
```

Comment:
Trying to figure out what type this should be -- I'd like it to take the request body as a parameter